ui/src/VoxettaMain.js

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+import {LitElement, html, css} from 'lit-element';
+
+export class VoxettaMain extends LitElement {
```

Comment:
You can define the custom element here

```javascript
import {blah, blah, customElement} from 'lit-element';

....

@customElement('vox-app')
export class VoxettaApp extends LitElement {

}
```

It looks like the above might only work in TypeScript, so you could do this in JS as well

```javascript
export class VoxettaApp extends LitElement {

}

customElements.defined('vox-app', VoxettaApp);
```

https://lit-element.polymer-project.org/guide/start


Edit: I see you already have the second option at the bottom. I suggest you pick a common prefix for all custom elements in your project, and stick to that throughout. Here you could use `vox-` as `voxetta` is kind of long. The component class name should follow the tag name.

For example

```javascript
export class VoxettaRecordButton
customElements.define('vox-record-button', VoxettaRecordButton)

export class VoxettaUserForm
customElements.define('vox-user-form', VoxettaUserForm)
```

Reply:
yup, @customElement only works in typescript. and yes, will use that prefix in the future.